tokio/src/runtime/blocking/pool.rs

```diff
@@ -86,18 +85,6 @@ where
     rt.spawn_blocking(func)
 }
 
-#[allow(dead_code)]
-pub(crate) fn try_spawn_blocking<F, R>(func: F) -> Result<(), ()>
```

Comment:
All usage of this code was removed a while ago, and this was forgotten because of the `allow`. I bumped into it since it was trying to spawn an un-instrumented future.